### DISCUSSION TITLE
什么时候能支持鸿蒙平台？

### DISCUSSION THREAD

**Discussion by dh3922** [Tencent/libpag#1635] [Q&A] [answered]
如题，什么时候能支持在鸿蒙原生APP上？

**Comment by kevingpqi123** [comment]
请问这里遇到的问题是啥，很多鸿蒙平台的 app 都在使用 pag

**Reply by dh3922** [reply]
您好，我意思是在基于原生鸿蒙平台开发的应用（是.hap应用，非.apk应用）时如何使用pag，引入方法、调用方法等等，是否能提供类似目前介绍里关于安卓、ios的说明文档。

**Reply by kevingpqi123** [reply, marked as answer]
由于我们自身没有类似的场景需求，目前人力有限，我们可能抽不出时间和精力来做这件事，建议你们自己适配，pag 是开源的，欢迎参与共建

**Reply by dh3922** [reply]
了解明白，那后面随着鸿蒙平台的发展你们的应用应该也会进行相对应的原生适配吧，所以应该肯定是会处理这个问题的，只是时间节点早晚的问题。希望到时候也能整理出来相应的文档，帮助广大的开发者在新平台上更好的使用pag。感谢。

**Reply by domchen** [reply]
鸿蒙Next 明年才会正式发布，到时候可以看看，应该主要处理一下编译相关的适配就行了。
